src/output.py

```diff
@@ -156,7 +156,7 @@ def composeFileCommand(command, lineObjs):
 
 
 def outputNothing():
-    appendToFile('echo "nothing to do!" && exit 1')
```

Comment:
If the shell does not provide a built-in echo function and external `echo` is not in $PATH. Very unlikely. And `fish` does not accept `A && B`. Thus I changed it to simply using `;`.